src/PHPDocker/Template/docker-compose-fragments/php-fpm.yml.twig

```diff
@@ -25,4 +25,5 @@
 {% if postgresContainerName %}        - postgres{{ "\n" }}{% endif %}
 {% if redisContainerName %}        - redis{{ "\n" }}{% endif %}
 {% if elasticsearchContainerName %}        - elasticsearch{{ "\n" }}{% endif %}
+{% if clickhouseContainerName %}        - clickhouse{{ "\n" }}{% endif %}
```

Comment:
:+1: One thing missing is an extra OR condition on the `if` statement on line 19, or clickhouse won't appear on links unless any of the other services is selected.

Mental note: refactor out that if statement to avoid precisely this situation.